### ISSUE TITLE
Having trouble bumping tags

### ISSUE THREAD

**Issue by robinjhuang** [mathieudutour/github-tag-action#46]
Hi,

Thank you for creating this Github action! In particular I like the prefix functionality that I couldn't find in other Github Actions. Having a little trouble getting the Semver bumps to work though.

Here's my Action workflow

```
- name: Bump version and push tag
        id: tag_version
        uses: mathieudutour/github-tag-action@v5
        with:
          github_token: ${{ secrets.GITHUB_TOKEN }}
          tag_prefix: graphqlv
          default_bump: patch
```

My last release was 0.1.0, and I pushed a commit with the message *fix(graphql): Remove Grpc code from graphql (#89)*. The Action analyzes the commit messages past the previous release and sees a previous commit: *feat(graphql): removed line from test*. 

So it decides to make a minor release, but still doesn't bump. I keep running into **Error Reference already exists.** Am I doing something wrong? 

```
* Remove protos
The release type for the commit is patch
Analyzing commit: feat(graphql): removed line from test
The release type for the commit is minor
Analyzing commit: fix(graphql): Remove proto
The release type for the commit is patch
Analyzing commit: remove grpc module
The commit should not trigger a release
Analyzing commit: fix(graphql): Remove Grpc code from graphql (#89)

* Remove grpc

* Fixed tests
The release type for the commit is patch
Analysis of 26 commits complete: minor release
New version is 0.1.0.
New tag after applying prefix is graphqlv0.1.0.
Changelog is # [0.1.0](https://github.com/mintothejoo/puffin/compare/v0.0.11...graphqlv0.1.0) (2020-12-08)

### Bug Fixes

* **graphql:** migrate remaining trip controller code ([#88](https://github.com/mintothejoo/puffin/issues/88)) ([c724396](https://github.com/mintothejoo/puffin/commit/c724396027d7953ce1b808a82844b1a783e4434d))
* **graphql:** Remove Grpc code from graphql ([#89](https://github.com/mintothejoo/puffin/issues/89)) ([04285cd](https://github.com/mintothejoo/puffin/commit/04285cd82741a29015718daf2ddbb6cd84af07b2))
* **graphql:** Remove proto ([8df386f](https://github.com/mintothejoo/puffin/commit/8df386f3651bac8b6bf87e2569db4404ecd7c2b5))


### Features

* **graphql:** removed line from test ([6cd5f0c](https://github.com/mintothejoo/puffin/commit/6cd5f0c95899704fa92bbc227ee77742dc97487c))

.
Error: Reference already exists

```

**Comment by andre487** [comment]
As I recognised, this action doesn't work with custom tag prefixes. It causes a "Reference already exists" error, but with default prefix "v" it works

**Comment by mathieudutour** [comment]
should be fixed on 5.1
